<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>19</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ac">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plus_minus">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_percent">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plus">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_multiplu">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>140</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>210</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>210</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_minus">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>210</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>280</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_division">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>280</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>280</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_equal">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>350</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dot">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>350</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_zero">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>291</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
